tests/integration/test_format_txts.py

```
@@ -0,0 +1,334 @@
+"""Integration tests for format_txts.py."""
+
+import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)